src/invoices/Voucher/Voucher.js

```diff
@@ -21,6 +22,17 @@ const Voucher = ({ match: { path } }) => (
         )
       }
     />
+    <Route
+      exact
+      path={`${path}:voucherId/edit`}
+      render={
+        props => (
+          <VoucherEditContainer
+            {...props}
+          />
+        )
+    }
+    />
```

Comment:
something wrong with formatting. 
Other thing - what the purpose of these 7 lines of code? Could it be shorter?